drakcore/package/Dockerfile

```diff
@@ -4,10 +4,13 @@ FROM $IMAGE
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
-    apt-get install -y wget curl python3.7 python3.7-venv python3-pip dh-virtualenv debhelper devscripts && \
+    apt-get install -y wget curl python3 python3-venv python3-pip debhelper devscripts && \
+    curl https://debs.icedev.pl/manual/focal-dev/dh-virtualenv_1.2~dev-1_all.deb -o dh-virtualenv.deb && \
```

Comment:
either install dh-virtualenv from repository or download our custom build if `"$(lsb_release -cs)" = "focal"` ?

seems to be only feasible option for now